src/studio/src/designer/backend.Tests/TestingControllers/DatamodelsControllerTests.cs

```diff
@@ -50,9 +50,9 @@ public async void Get_Datamodel_Ok()
             string responsestring = await response.Content.ReadAsStringAsync();
             TextReader textReader = new StringReader(responsestring);
             JsonValue jsonValue = await JsonValue.ParseAsync(textReader);
-            JsonSchema jsonSchema = new Manatee.Json.Serialization.JsonSerializer().Deserialize<JsonSchema>(jsonValue);
+
+            new JsonSerializer().Deserialize<JsonSchema>(jsonValue);
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-            Assert.Equal(5, jsonSchema.Count);
```

Comment:
Was this Assert ment to be removed permanently?

Reply:
Yes. It simply tests that a mock is able to retrieve the correct file, but not any significant logic.